config/README.md

```diff
@@ -125,7 +125,32 @@ _your own configuration lines_
 
 ### `.github/workflows/meta.yml`
 
-_At this time there are no configuration options_.
+Add the `[github]` TOML table in `.meta.toml`,
+and set the enabled jobs with the `jobs` key.
+
+```toml
+[github]
+jobs = [
+    "qa",
+    "test",
+    "coverage",
+    "dependencies",
+    "release-ready",
```

Comment:
```suggestion
    "release_ready",
```

Let's use the actual name, so we can simplify the template, there's already way too many (and more to come) options to remember how things should be named.

Maybe we can mention that the complete list is on https://github.com/plone/meta/tree/main/.github/workflows ? 🤔